main.go

```diff
@@ -30,6 +30,7 @@ var (
 		"extraDNSRecords": handleArray(),
 		"grants":          handleArray(),
 		"groups":          handleObject(),
+		"ipsets":          handleArray(),
```

Comment:
`ipsets` [is an object](https://tailscale.com/kb/1387/ipsets#syntax), not an array, so it should use `handleObject()` instead. Can you make that change and also update [testdata](https://github.com/tailscale-dev/tailscale-acl-combiner/tree/main/testdata) to ensure tests pass?

Reply:
Yup -- I guess that is my derpy mistake; I was just skimming `handleArray()` and `handleObject()` and I thought it was about recombination of the values for cases where the keys match.

Will update.